modifiers.lisp

```diff
@@ -3,13 +3,13 @@
 
 (in-package :nkeymaps/modifier)
 
-(defparameter +control+ (nkeymaps:define-modifier :string "control" :shortcut "C")
+(defparameter +control+ (define-modifier :string "control" :shortcut "C")
   "The key modifier commonly known as 'control'.")
 (defparameter +meta+ (define-modifier :string "meta" :shortcut "M")
-  "The key modifier commonly known as 'meta' or 'alt'.")
+  "The key modifier commonly known as 'alt' or 'option'.")
```

Comment:
I'd retain `meta` mention here, because that's still a vibrant echo of the times long gone 😛